.github/workflows/update-mozilla-ca-cert.yaml

```diff
@@ -0,0 +1,35 @@
+name: Update Mozilla cert.pem Nightly
+
+on:
+  schedule:
+    - cron: "0 11 * * *"
+  workflow_dispatch:
+
+permissions:
+  contents: write
+  pull-requests: write
+
+concurrency:
+  group: ${{ github.ref }}-${{ github.workflow }}-${{ github.event_name }}
+  cancel-in-progress: true
+
+jobs:
+  update_cert:
+    name: Update Mozilla cert.pem Nightly
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Download latest Mozilla CA cert.pem
+        run: |
+          curl --silent --show-error --location https://curl.se/ca/cacert.pem -o chia/ssl/cacert.pem
+
+      - name: Create Pull Request if cacert.pem changed
+        uses: peter-evans/create-pull-request@v6
+        with:
+          commit-message: "Update cacert.pem from Mozilla CA bundle"
+          title: "Update cacert.pem from Mozilla CA bundle"
+          body: "Automated update of cacert.pem from https://curl.se/ca/cacert.pem."
+          branch: update-cacert-pem
+          add-paths: chia/ssl/cacert.pem
```

Comment:
before this step, we should probably grab the key and set up commit signing, so we'll be able to merge this PR once created. Example action to use in the current mozilla ca repo
https://github.com/Chia-Network/mozilla-ca/blob/main/.github/workflows/update-cert.yaml#L33:L37